targets/science/include/PinNames.h

```diff
@@ -165,6 +167,11 @@ typedef enum {
   BUTTON_1 = PB_0,
   BUTTON_2 = PB_1,
 
+  /**** Servo Pins *****/
+
+  SRVO_PWM_1 = PB_2,
+  SRVO_PWM_2 = PB_10,
+
```

Comment:
did u add this?

Reply:
no this was there from the beginning, I can remove it I dont think it does anything

Reply:
see below

Reply:
sounds good